test_files/fields/fields.ts

```diff
@@ -22,3 +22,17 @@ fieldsTest.field1 = 'hi';
 let AnonymousClass = class {
   field: number;
 };
+
+
+class BaseThatThrows {
+  get throwMe(): number { throw new Error(); }
```

Comment:
have you tried and seen this fail? I'm always unsure on whether this happens if you declare it as as static or as an instance property.

Reply:
Yep, I produced this case by verifying it throws.  Just to double-check I pasted the JS code in the test case into a a JS repl and verified that it throws if I remove the function wrapper.